tests/utils/worker_utils.cpp

```diff
@@ -155,6 +155,10 @@ void execFuncWithPool(faabric::Message& call, bool clean, int timeout)
     faabric::Message result = sch.getFunctionResult(call.id(), 30000);
     REQUIRE(result.returnvalue() == 0);
 
+    if (result.outputdata().size() > 0) {
```

Comment:
I know there's a `execFuncWithStringResult`, but it felt unnatural to use it. And it also felt unnatural to pass an optional parameter to print the `outputdata`. I think printing it always as long as its got something in it (`results.outputdata()`, I mean), is acceptable.

Reply:
The output isn't always human-redable, so this may pollute the logs with stuff that's hard to decipher. 

If you've found it useful though it's probably worth keeping, but:

- Make it at least debug level if not trace
- Prefix it with `Function <function_id> output: ` so that it doesn't look like junk

Reply:
You are right. I will delete this.